CoreScriptsRoot/Modules/Settings/Pages/Players.lua

```diff
@@ -274,7 +274,7 @@ local function Initialize()
 					table.insert(existingPlayerLabels, index, frame)
 				end
 				frame.Name = 'PlayerLabel'..player.Name
-				frame.Icon.Image = 'http://www.roblox.com/Thumbs/Avatar.ashx?x=100&y=100&userId='..math.max(1, player.userId)
+				frame.Icon.Image = 'http://www.roblox.com/Thumbs/Avatar.ashx?x=100&y=100&userId='..tostring(player.userId)
```

Comment:
The friendStatusCreate function makes it so if the UserId is under 1 then it doesn't create a 'Add Friend'. I ran a test server to make sure this would work. If the UserId is under 1 then it goes to a default image. Note this was ran in "Test" so that the UserId would be -1. As you can see here is the image set to users that Id is under -1, Guests, Studio, etc. http://prntscr.com/b9arpd I am thinking the website returns that image if the UserId is under -1. So this could technically work.